app/views/transfer/new.html.erb

```diff
@@ -1 +1,100 @@
 <%= content_for(:title, "Thesis Transfer Submission | MIT Libraries") %>
+
+<% content_for :additional_js do %>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.17.0/jquery.validate.min.js"></script>
+<% end %>
+
+<%= simple_form_for(@transfer, url: '/transfer', html: { class: 'transferSubmission' }, validate: true ) do |f| %>
+
+  <%= f.error_notification %>
+  <div class="alert alert-banner error" style="display: none;" role="alert" aria-invalid="true"></div>
+
+  <%= f.association :department, label_method: :name, as: :select,
+        collection: current_user.departments.order(:name),
```

Comment:
I think this should be the default for this field, but for `theses_admins` I believe it should just display _all_ Departments. If that is a pain to make that change (and associated tests) I would be okay with opening a specific ticket to address that distinction. The current solution will technically not restrict admins from submitting, but the form will never allow them to choose anything for this field so effectively they still can't submit :)

Reply:
Hmm... ok. I had thought that, for users with the `:thesis_admin` ability, this call would end up returning all departments because their ability would be for all departments. If we want to structure this differently, that's fine - I can make this change and its test.

Reply:
The way it's modeled we need to handle that in the UI here or would need more complex behavior in the models. I normally move towards complexity in the models so maybe we should talk about your expectation as it may be a very good one.

Reply:
I think this is resolved now...